build_ngtf.py

```diff
@@ -150,6 +150,12 @@ def main():
         help="Builds and links ngraph statically\n",
         action="store_true")
 
+    # parser.add_argument(
+    #     '--use_parallel_executor',
+    #     help=
+    #     "Uses the parallel Executor that speeds up the inference use cases for certain backends.\n",
+    #     action="store_true")
+
```

Comment:
Commented code

Reply:
@kanvi-nervana  These avijit's changes of Parallel Executor, Please comment on his PR,
I only added 3 files Ngraph_executor_dh.h, Ngraph_executor_dh.cpp, and test_executor.cpp